api/src/main/resources/AppointmentStatusHistory.hbm.xml

```diff
@@ -7,7 +7,9 @@
 
 	<class name="AppointmentStatusHistory" table="${project.parent.artifactId}_appointment_status_history">
 		<id name="appointmentStatusHistoryId" type="int" column="appointment_status_history_id" unsaved-value="0">
-			<generator class="native" />
+			<generator class="native">
+				<param name="sequence">appointmentscheduling_appoint_appointment_status_history_id_seq</param>
```

Comment:
Can you include comments for those names you are shortening? Stating what the correct name should have been and why you are making it shortened.

Reply:
There is a problem in actual length of sequence that can be stored in PostgreSQL. When length excedes the limit then table name gets stripped from end.

Here there is stripping in below sequences :

1. Original Sequence Name- appointmentscheduling_appointment_block_appointment_block_id_seq

Current name as set in AppointmentBlock.hbm file - appointmentscheduling_appointment_bloc_appointment_block_id_seq

 

2. Original Sequence Name- appointmentscheduling_appointment_request_appointment_request_id_seq

Current name as set in AppointmentRequest.hbm file - appointmentscheduling_appointment_re_appointment_request_id_seq

 

3. Original Sequence Name-appointmentscheduling_appointment_status_history_appointment_status_history_id_seq

Current name as set in AppointmentStatusHistory.hbm file - appointmentscheduling_appoint_appointment_status_history_id_seq

 

4. Original Sequence Name-appointmentscheduling_provider_schedule_provider_schedule_id_seq

Current name as set in ProviderSchedule.hbm file - appointmentscheduling_provider_schedul_provider_schedule_id_seq

Reply:
I mean that we should include the comment in the code itself. So when one wants to correct it, they see the comment explaining that we did not make it so by mistake.

Reply:
Oh, understood.

Reply:
@dkayiwa , I have added the comments in code.